packages/iris-grid/src/sidebar/visibility-ordering-builder/sortable-tree/TreeItem.module.scss

```diff
@@ -0,0 +1,162 @@
+@import '@deephaven/components/scss/custom.scss';
+
+.Wrapper {
```

Comment:
class names are normally kebab-case by convention. Did you do it this way because you are using it as a module?

Reply:
I did it this way because I copied it from dnd-kit. I'll clean up to our conventions